package.json

```diff
@@ -2,30 +2,30 @@
   "name": "matanuska",
   "version": "0.0.1",
   "description": "A chill BASIC to shell to",
-  "main": "dist/index.js",
+  "type": "module",
+  "main": "dist/index.mjs",
   "bin": {
     "matbas": "bin/matbas"
   },
   "scripts": {
     "install:citree": "cd tools/citree && npm install",
     "install:test-generator": "cd tools/test-generator && npm install",
     "build:config": "./scripts/build-config.mjs",
-    "build": "concurrently 'npm run build:templates' 'npm run build:ast' && npm run build:swc",
+    "build": "concurrently 'npm run build:templates' 'npm run build:ast' && npm run build:vite",
     "build:release": "dotenv -e release.env -- npm run build",
     "build:constants": "dotenv -- ./scripts/constants.sh",
     "build:debug": "dotenv -- ./scripts/debug.sh",
     "build:templates": "concurrently 'npm run build:constants' 'npm run build:debug'",
-    "build:tsc": "tsc -p tsconfig.build.json",
-    "build:swc": "swc --config-file .swcrc.build -d dist *.ts **/*.ts",
+    "build:vite": "vite build",
     "build:citree": "cp ast/index.citree tools/citree/example/ast.citree && cd tools/citree && npm run build && npm run snap",
     "build:test-generator": "cd tools/test-generator && npm run build",
     "build:ast": "citree ast/index.citree",
     "build:test": "npm run build:test-generator && dotenv -e test.env -- test-generator ./test",
     "build:test:activate": "dotenv -e test.env -- test-generator --activate ./test",
     "check": "dotenv -e test.env -- concurrently 'npm run build:templates' 'npm run build:ast' && tsc --noEmit",
-    "format": "prettier --write \"**/*.ts\" \"*.mjs\" \"scripts/*.mjs\"",
+    "format": "prettier --write \"**/*.ts\" \"**/*.mjs\"",
     "start": "npm run build && PATH=\"${PATH}:$(pwd)/bin\" bin/matbas",
-    "lint": "dotenv -e test.env -- npm run build && prettier --list-different \"**.ts\" && eslint --fix \"**.ts\"",
+    "lint": "dotenv -e test.env -- npm run build:templates && prettier --list-different \"**.ts\" && eslint --fix \"**.ts\"",
```

Comment:
Last note, I should get the build scripts to do up prettier and eslint configs too.